src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -1,13 +1,46 @@
 package org.teamresistance.frc;
 
 import edu.wpi.first.wpilibj.IterativeRobot;
+import org.strongback.Strongback;
 
 /**
  * Main robot class. Override methods from {@link IterativeRobot} to define behavior.
+ *
+ * @author Shreya Ravi
  */
 public class Robot extends IterativeRobot {
 
-    @Override
-    public void robotInit() {
-    }
+  private final Drive drive = new Drive(IO.robotDrive, IO.leftJoystick.getRoll(), IO.leftJoystick.getPitch(), IO.rightJoystick.getRoll());
```

Comment:
so if i'm splitting up a line because there are too many parameters, i tend to like with every parameter on its own line like this:
```
private final Drive drive = new Drive(
    IO.robotDrive,
    IO.leftJoystick.getRoll(), 
    IO.leftJoystick.getPitch(),
    IO.rightJoystick.getRoll()
);
```
Let me know what you think.